tests/settings/e2e/steps/stakepool-settings.js

```diff
@@ -0,0 +1,79 @@
+import { When, Then } from 'cucumber';
+
+const CUSTOM_SERVER_DROPDOWN_OPTION = '//*[@class="SimpleInput_customValueWrapper"]//*[@value="Custom server"]';
+const DAEDALUS_TOP_BAR_LOGO = '//*[@class="TopBar_topBar TopBar_withoutWallet"]';
+const NOT_A_VALID_SMASH_SERVER_ERROR_MESSAGE_ACTIVATED = '//*[@class="InlineEditingInput_errorMessage" and text()="This URL is not a valid SMASH server"]';
+const OFF_CHAIN_METADATA_SERVER_SMASH_LABEL = '//*[contains(text(), "Off-chain metadata server (SMASH)")]';
+const SMASH_SERVER_URL_INPUT_BOX = '//*[@label="SMASH server URL"]';
+const STAKE_POOL_CUSTOM_SERVER_INPUT_BOX_SUBMIT_BUTTON = '//*[@class="InlineEditingInput_button InlineEditingInput_okButton SimpleButton_root ButtonOverrides_root"]';
+const STAKE_POOL_CUSTOM_SERVER_INPUT_BOX_X_BUTTON = '//*[@class="InlineEditingInput_button InlineEditingInput_cancelButton SimpleButton_root ButtonOverrides_root"]';
+const STAKE_POOL_SERVER_DROPDOWN = '//*[@class="SimpleFormField_inputWrapper"]';
+const STAKE_POOL_SERVER_DROPDOWN_CUSTOM_OPTION = '//*[@class="ScrollbarsCustom-Content"]//span[text()="Custom server"]';
+const STAKE_POOLS_SUBMENU_SETTINGS = '//*[@class="SettingsMenu_component"]//button[text()="Stake pools"]';
+
+When(/^custom server is the default option$/, function() {
+  return this.waitAndClick(CUSTOM_SERVER_DROPDOWN_OPTION);
+});
+
+When(/^I clicked outside of the input-box to change focus$/, function() {
+  // This step was necessary as when the error message for this box is displayed the submit button could not receive the click and i got a "Element is not clickable at point error"
+  return this.waitAndClick(DAEDALUS_TOP_BAR_LOGO);
+});
+
+Then(/^The custom server I chose earlier is visible on stake-pool screen above stake-pool list and is clickable$/, function() {
+  const serverUrl = "https://smash.cardano-testnet.iohkdev.io/";
```

Comment:
Better to have this as a constant, among the others on the top, using the snake case (SERVER_URL)

Reply:
Fixed 👍